### PR TITLE
[Py] Added Dequantize, Quantize, Quantized Convolution, Quantized Dot…

### PR DESCRIPTION
**Pull request by Ewa21**
… operators to Python API.

### CODE REVIEW

File: python/ngraph/ops.py [hunk 1]
```diff
@@ -168,6 +170,190 @@ def mvn(data, axes, normalize_variance, eps, name=None):
...
 
 
+@nameable_op
+def quantize(data, scale, zero_point, new_type, axes, round_mode, name=None):
+    # type: (Node, Node, Node, NumericType, Set[int], int, str) -> Node
+    r"""Perform quantize operation on data from input node.
+
+    Computes quantize on the input tensor:
+
+    .. math:: output = ROUND((input /scale) + zero\_point)
```
Comment (tsocha) [inline python/ngraph/ops.py]:
```suggestion
    .. math:: output = ROUND((input / scale) + zero\_point)
```

File: python/ngraph/ops.py [hunk 2]
```diff
@@ -168,6 +170,190 @@ def mvn(data, axes, normalize_variance, eps, name=None):
     return MVN(data, AxisSet(axes), normalize_variance, eps)
 
 
+@nameable_op
+def quantize(data, scale, zero_point, new_type, axes, round_mode, name=None):
+    # type: (Node, Node, Node, NumericType, Set[int], int, str) -> Node
```
Comment (postrational) [inline python/ngraph/ops.py]: The `round_mode` should not be marked as `int` type but rather the enum type you created.

File: python/ngraph/ops.py [hunk 3]
```diff
@@ -168,6 +170,190 @@ def mvn(data, axes, normalize_variance, eps, name=None):
...
+
+    Computes quantize on the input tensor:
+
+    .. math:: output = ROUND((input / scale) + zero\_point)
+
+    :param data: The node with data tensor.
+    :param scale: Scale used for mapping.
+    :param zero_point: Zero point used for mapping.
+    :param new_type: Output element type.
+    :param axes: Number describes how to perform ROUND function.
```
Comment (postrational) [inline python/ngraph/ops.py]: This documentation should be using the names of the modes, not integer values.
Also, is this for `axes` or `round_mode`?

Comment (Ewa21) [reply python/ngraph/ops.py]: Ok. But shouldn't it provide informations about which number you should use to obtain a certain type of `ROUND`?

**Pull request merged**